gtsfm/loader/colmap_loader.py

```diff
@@ -83,16 +83,32 @@ def __init__(
             self._calibrations = self._calibrations * len(img_fnames)
 
         # preserve COLMAP ordering of images
+        
+        self._img_fnames = []
         self._image_paths = []
-        for img_fname in img_fnames:
+        self._wTi_list = []
+        
+        # If one of the images is not found on disk, the assigned image indices will be re-ordered on disk
+        # to skip the missing image.
+        for img_fname, wTi in zip(img_fnames, wTi_list):
             img_fpath = os.path.join(images_dir, img_fname)
             if not Path(img_fpath).exists():
                 continue
+            self._img_fnames.append(img_fname)
             self._image_paths.append(img_fpath)
+            self._wTi_list.append(wTi)
 
         self._num_imgs = len(self._image_paths)
         logger.info("Colmap image loader found and loaded %d images", self._num_imgs)
 
+    def get_image_fname(self, idx: int) -> str:
```

Comment:
unused function?

Reply:
Thanks for taking a look @akshay-krishnan. I forgot to mention I'm using these for testing in another branch.